gitapp/lib/users_list/users_list.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:gitapp/users_list/user.dart';
+//Lists of the users FGA-3
+class UsersList {
+  List<User> userlist= List.empty(growable: true);
+  List<User> searchedlist= List.empty(growable: true);
+  UsersList(){
+    for(int i=0;i<5;i++)
+    {
+
+      User user = new User(
+          avatarUrl: 'https://i.kym-cdn.com/entries/icons/original/000/035/310/Peepo_Animation_Banner.jpg',
+          username: "User"+i.toString());
+      userlist.add(user);
+    }
+
+  }
+
+void createUsers()  {
+  userlist.clear();
+for(int i=0;i<5;i++)
+  {
+
+    User user = new User(
+        avatarUrl: 'https://i.kym-cdn.com/entries/icons/original/000/035/310/Peepo_Animation_Banner.jpg',
+        username: "User"+i.toString());
+    userlist.add(user);
+  }
+
+
+}
+
+  void GetSearchedUsers(String text)
```

Comment:
name of the method has to start from lowercased letter.
Also if you are saying `get` then it suggests that the method should return something, but in fact it does not.

Reply:
solved